### DISCUSSION TITLE
How can I fetch the descriptions of trending videos and use them as filenames?

### DISCUSSION THREAD

**Discussion by Ckatalay** [davidteather/TikTok-Api#532] [Q&A] [answered]
I managed to download videos but I couldn't find how can I fetch descriptions on the documentation

**Comment by alzeric** [comment, marked as answer]
Descriptions are unreliable from my experience (usually filled with silly emojis or sometimes no description at all).  You'd have to write a hefty user sanitization function to make anything from the description usable.

I simply use the video_id as the filename and it works out great.  If you want the Descriptions along with the videos I'd log the descriptions along with the video_id in a json log or something along those lines you can use elsewhere.

**Reply by davidteather** [reply]
Yeah, I agree saving by video ids is the best solution. Video descriptions should be under the 'desc' key if I'm remembering correctly and if TikTok hasn't changed its structure too much.

**Reply by Ckatalay** [reply]
Ok I'll use video IDs thanks
